components/profile_needs_selector.js

```diff
@@ -60,15 +77,36 @@ export class ProfileNeedsSelector extends Component {
       <ExpansionPanel
         className={classnames(classes.root)}
         defaultExpanded
-        disabled={pageWidth >= 600 ? true : false}
         expanded={pageWidth >= 600 ? true : this.state.open}
       >
         <ExpansionPanelSummary
           className={classnames(classes.summary)}
           expandIcon={pageWidth >= 600 ? "" : <ExpandMoreIcon />}
           onClick={pageWidth >= 600 ? foo => foo : () => this.toggleOpenState()}
         >
-          <Typography variant="title">{t("filters")}</Typography>
+          <Typography
+            variant="title"
+            className={classnames(classes.filterTitle)}
+          >
+            {t("filters")}{" "}
+            {(JSON.stringify(this.props.selectedNeeds) !== "{}" ||
+              this.props.patronType !== "") &&
+            pageWidth > 600 ? (
+              <Button
```

Comment:
I played around with making this an <a>, which makes it small enough to not overflow in french. Maybe we can try that in a later PR.